pypeit/onespec.py

```diff
@@ -118,14 +120,14 @@ def _bundle(self):
         """
         return super()._bundle(ext='SPECTRUM')
 
-    @property
-    def sig(self):
-        """ Return the 1-sigma array
-
-        Returns:
-            `numpy.ndarray`_: error array
-        """
-        return np.sqrt(utils.inverse(self.ivar))
+    #@property
```

Comment:
Probably this commented section should be removed, if sigma is now stored in the DataModel

Reply:
Done.